pretext/ExtraTopics/More-BigO.ptx

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-more-bigo">
+  <title>Big O Simplification and Practice</title>
+
+  <p>One more important topic in Big O Analysis is <term>simplification</term>. You will often be asked to do
```

Comment:
"You will often be asked to do this on assignments". Not really true, and I'm not sure we'll even have time to do as much Big O as last time with the badly-placed holiday. I'd remove that mention.